noir-projects/noir-contracts/contracts/token_contract/src/types/token_note.nr

```diff
@@ -48,6 +55,21 @@ impl NoteInterface<TOKEN_NOTE_LEN, TOKEN_NOTE_BYTES_LEN> for TokenNote {
         ]);
         (note_hash_for_nullify, nullifier)
     }
+
```

Comment:
I've manually implemented this function here because of https://github.com/AztecProtocol/aztec-packages/issues/7738